requirements.txt

```diff
@@ -6,9 +6,8 @@ opencv-contrib-python==4.5.4.58 ; platform_machine != "aarch64" and platform_mac
 opencv-contrib-python==4.5.1.48 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version != "3.10"
 opencv-python==4.4.0.46 ; platform_machine == "armv6l" or platform_machine == "armv7l"
 opencv-contrib-python==4.4.0.46 ; platform_machine == "armv6l" or platform_machine == "armv7l"
--e ./depthai_sdk
+depthai-sdk
 --extra-index-url https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/wheels/
-pyqt5>5,<5.15.6 ; platform_machine != "armv6l" and platform_machine != "armv7l" and platform_machine != "aarch64"
```

Comment:
CC: @daniilpastukhov @Erol444

Reply:
> Have you thought about changing the dependency package `pyside2` of `depthai_sdk` and the dependency package `pyqt5` of `depthai_demo` to the same one?

Great idea. I will change the demo demendency to pyside2

Reply:
Don't we need pyqt5 to run depthai_demo.py? I support depthai_sdk has pyqt5 in its requirements, but we will remove them in the upcoming merges. It'd be better to keep it here IMO.

Reply:
Yes, pyqt5 should stay (preferably here) - mostly wondered for the `-e ./depthai-sdk` change

Reply:
I think editable mode isn't necessary (e.g. in case smb is pulling the updates and doesn't want `depthai_sdk` to update). Also it's more convenient for controlling what version is installed.

Reply:
I think the `-e ./depthai-sdk` is not necessary, if you think it is better the way it is, I will keep it like this. 

to pyqt5 i will add platform_machine != arm64 so that it is not installed on mac silicon